src/core/meshcop/commissioner.cpp

```diff
@@ -916,8 +964,9 @@ void Commissioner::SendJoinFinalizeResponse(const Coap::Header &aRequestHeader,
     mNetif.GetDtls().Send(buf, static_cast<uint16_t>(cur - buf));
     mSendKek = false;
 
-    otLogInfoMeshCoP("sent joiner finalize response\r\n");
-    otLogCertMeshCoP("[THCI] direction=send | type=JOIN_FIN.rsp\r\n");
+    otLogInfoMeshCoP("sent joiner finalize response\n");
+    otLogCertMeshCoP("[THCI] direction=send | type=JOIN_FIN.rsp\n");
```

Comment:
Please keep "\r" in all of the logs to ensure the next log starts from the first column of a new line (It will not start from the first column without "\r" on CC2538 platform).